relations/src/proof.rs

```diff
@@ -0,0 +1,102 @@
+use halo2_base::{
```

Comment:
If this file is about Merkle proofs, I think it's better to call it `merkle_proof`

Reply:
Done